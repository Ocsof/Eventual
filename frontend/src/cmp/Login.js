import React from 'react';
import {Link} from "react-router-dom";
import Home from "./Home";

class Login extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            email: '',
            password: '',
            isLogged: false
        };

        this.handleEmailChange = this.handleEmailChange.bind(this);
        this.handlePasswordChange = this.handlePasswordChange.bind(this);
        this.handleSubmit = this.handleSubmit.bind(this);
    }

    handleEmailChange(event) {
        this.setState({email: event.target.value});
    }

    handlePasswordChange(event) {
        this.setState({password: event.target.value});
    }

    handleSubmit(event) {
        if( this.state.password !== '' && this.state.email !== ''){
            /*
                TODO: change with control function on database subscribes
             */
            alert('Login effettuato con successo: ' + this.state.email);
            this.setState({
                isLogged: true
            })
        } else {
            alert('Email o password non corretta');
        }
        event.preventDefault();
    }

    render() {
        if(!this.state.isLogged) {
            return (
            <div>
                <form onSubmit={this.handleSubmit}>
                    <label>
                        Email:
                        <input type="text" alt="email" autoComplete="email" value={this.state.email} onChange={this.handleEmailChange} />
                    </label>
                    <br/>
                    <label>
                        Password:
                        <input type="password" alt="password" autoComplete="current-password" value={this.state.password} onChange={this.handlePasswordChange} />
                    </label>
                    <br/>
                    <input type="submit" value="Submit" />
                </form>
                <Link to="/passwordRecover">Ho dimenticato la password</Link>
                <br/>
                <Link to="/signup">Clicca qui se non sei ancora iscritto</Link>
            </div>
            )
        } else {
            return (
                <div>
                    <Home />
                </div>
            )
        }
    }
}

export default Login;