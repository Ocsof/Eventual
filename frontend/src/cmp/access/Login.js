import React from 'react';
import {Link} from "react-router-dom";

class Login extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            email: '',
            password: '',
            isLogged: false
        };

        this.handleEmailChange = this.handleEmailChange.bind(this);
        this.handlePasswordChange = this.handlePasswordChange.bind(this);
        this.handleSubmit = this.handleSubmit.bind(this);
    }

    handleEmailChange(event) {
        this.setState({email: event.target.value});
    }

    handlePasswordChange(event) {
        this.setState({password: event.target.value});
    }

    handleSubmit(event) {
        if( this.state.password !== '' && this.state.email !== ''){
            /*
                TODO: change with control function on database subscribes
            */
            alert('Login effettuato con successo: ' + this.state.email);
            this.setState({
                isLogged: true
            })
        } else {
            alert('Email o password non corretta');
        }
        event.preventDefault();
    }

    render() {
        return (
            <section className="gradient-custom">
                <div className="container-fluid h-custom">
                    <div className="row d-flex justify-content-center align-items-center h-100">
                        <div className="col-md-9 col-lg-6 col-xl-5">
                            <img src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-login-form/draw2.webp"
                                 className="img-fluid" alt="Sample image" />
                        </div>
                        <div className="col-md-8 col-lg-6 col-xl-4 offset-xl-1">
                            <form onSubmit={this.handleSubmit}>
                                <div
                                    className="d-flex flex-row align-items-center justify-content-center justify-content-lg-start">
                                    <p className="lead fw-normal mb-0 me-3">Sign in with</p>
                                    <a href="https://www.facebook.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle">
                                        <i className="fab fa-facebook-f"></i>
                                    </a>

                                    <a href="https://www.twitter.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle">
                                        <i className="fab fa-twitter"></i>
                                    </a>

                                    <a href="https://www.linkedin.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle">
                                        <i className="fab fa-linkedin-in"></i>
                                    </a>
                                    <a href="https://www.google.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle  ">
                                        <i className="fab fa-google"></i>
                                    </a>
                                </div>

                                <div className="divider d-flex align-items-center my-4">
                                    <p className="text-center fw-bold mx-3 mb-0">Or</p>
                                </div>

                                <div className="form-outline mb-4">
                                    <input type="email" id="form3Example3" className="form-control form-control-lg"
                                           placeholder="Enter a valid email address" onChange={this.handleEmailChange}/>
                                    <label className="form-label" htmlFor="form3Example3">Email address</label>
                                </div>

                                <div className="form-outline mb-3">
                                    <input type="password" id="form3Example4" className="form-control form-control-lg"
                                           placeholder="Enter password" onChange={this.handlePasswordChange}/>
                                    <label className="form-label" htmlFor="form3Example4">Password</label>
                                </div>

                                <div className="d-flex justify-content-between align-items-center">
                                    <div className="form-check mb-0">
                                        <input className="form-check-input me-2" type="checkbox" value=""
                                               id="form2Example3"/>
                                        <label className="form-check-label" htmlFor="form2Example3">
                                            Remember me
                                        </label>
                                    </div>
                                    <Link to="/passwordRecover" className="link-danger small">Forgot password?</Link>
                                </div>

                                <div className="text-center text-lg-start mt-4 pt-2">
                                    <button type="submit" className="btn btn-primary btn-lg"
                                            style={{paddingLeft: "2.5rem", paddingRight: "2.5rem"}} >Login
                                    </button>
                                    <p className="small fw-bold mt-2 pt-1 mb-0">Don't have an account? <Link to="/signup"
                                                                                                          className="text-body">Register</Link>
                                    </p>
                                </div>

                            </form>
                        </div>
                    </div>
                </div>
            </section>
        )
    }
}

export default Login;