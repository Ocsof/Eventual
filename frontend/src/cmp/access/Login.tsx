import React, {useState} from 'react';
import {Link, useNavigate} from "react-router-dom";
import {NotificationManager} from "react-notifications";
import {useLogin} from "./LoginContext";
import {Button} from "react-bootstrap";

export function Login(){
    const navigate = useNavigate();
    const {toggleLogin} = useLogin();
    const isLoggedIn = Boolean(localStorage.getItem("logged"));
    const [user, setUser] = useState({
        email: "",
        password: ""
    });

    function handleSubmit(e) {
        e.preventDefault()
        /*todo: change with check in the database*/
        if(user.email !== "" && user.password !== ""){
            NotificationManager.success("Benvenuto: " + user.email)
            toggleLogin()
            localStorage.setItem('username', user.email)
            localStorage.setItem('logged', String(isLoggedIn))
            navigate("/events")
        } else {
            NotificationManager.error('Email o password non corretta');
        }
    }

    function handleLogout(e) {
        e.preventDefault()
        NotificationManager.success("Logout")
        toggleLogin()
        localStorage.removeItem('logged')
    }

    return (
        isLoggedIn ? (
            <>
                <section className="gradient-custom w-75">
                    <div className="container-fluid h-custom">
                        <div className="row d-flex justify-content-center align-items-center h-100">
                            <div className="col-md-9 col-lg-6 col-xl-5">
                                <img src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-registration/draw1.webp"
                                    className="img-fluid" alt="Sample logout" />
                            </div>
                        </div>
                    </div>
                    <div className="row d-flex justify-content-center align-items-center">
                        <h3 className="row d-flex justify-content-center align-items-center">Benvenuto: {localStorage.getItem('username')}</h3>
                        <Button className="btn btn-primary btn-lg m-2" style={{width:"50%"}} onClick={() => navigate("/edit_profile")}>Edit profile <i className="fa-solid fa-user-pen"></i></Button>
                        <Button className="btn btn-primary btn-lg m-2" style={{width:"50%"}} onClick={(e) => handleLogout(e)}>Logout <i className="fa-solid fa-right-from-bracket" /></Button>
                    </div>
                </section>
            </>
        ): (
            <section className="gradient-custom w-100">
                <div className="container-fluid h-custom">
                    <div className="row d-flex justify-content-center align-items-center h-100">
                        <div className="col-md-9 col-lg-6 col-xl-5">
                            <img src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-login-form/draw2.svg"
                                 className="img-fluid" alt="Sample login" />
                        </div>
                        <div className="col-md-8 col-lg-6 col-xl-4 offset-xl-1">
                            <form onSubmit={e => handleSubmit(e)} autoComplete="current-password">
                                <div
                                    className="d-flex flex-row align-items-center justify-content-center justify-content-lg-start">
                                    <p className="lead fw-normal mb-0 me-3">Sign in with</p>
                                    <a href="https://www.facebook.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle">
                                        <i className="fab fa-facebook-f"></i>
                                    </a>

                                    <a href="https://www.twitter.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle">
                                        <i className="fab fa-twitter"></i>
                                    </a>

                                    <a href="https://www.linkedin.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle">
                                        <i className="fab fa-linkedin-in"></i>
                                    </a>
                                    <a href="https://www.google.com" type="button" className="btn btn-primary btn-floating mx-1 rounded-circle  ">
                                        <i className="fab fa-google"></i>
                                    </a>
                                </div>

                                <div className="divider d-flex align-items-center my-4">
                                    <p className="text-center fw-bold mx-3 mb-0">Or</p>
                                </div>

                                <div className="form-outline mb-4">
                                    <input type="email" id="loginEmail" className="form-control form-control-lg"
                                           placeholder="Enter a valid email address" onChange={(e) => setUser({...user, email: e.target.value})} value={user.email}/>
                                    <label className="form-label" htmlFor="loginEmail">Email address</label>
                                </div>

                                <div className="form-outline mb-3">
                                    <input type="password" id="loginPw" className="form-control form-control-lg"
                                           placeholder="Enter password" onChange={(e) => setUser({...user, password: e.target.value})} value={user.password}/>
                                    <label className="form-label" htmlFor="loginPw" >Password</label>
                                </div>

                                <div className="d-flex justify-content-between align-items-center">
                                    <div className="form-check mb-0">
                                        <input className="form-check-input me-2" type="checkbox" value=""
                                               id="remember"/>
                                        <label className="form-check-label small" htmlFor="remember">Remember me</label>
                                    </div>
                                    <Link to="/passwordRecover" className="link-danger small">Forgot password?</Link>
                                </div>

                                <div className="text-center text-lg-start mt-4 pt-2">
                                    <button type="submit" className="btn btn-primary btn-lg" style={{paddingLeft: "2.5rem", paddingRight: "2.5rem"}} >
                                        Login
                                    </button>
                                    <p className="small fw-bold mt-2 pt-1 mb-0">Don't have an account?
                                        <Link to="/signup" className="text-body mx-1">Register</Link>
                                    </p>
                                </div>
                            </form>
                        </div>
                    </div>
                </div>
            </section>
        )
    )
}