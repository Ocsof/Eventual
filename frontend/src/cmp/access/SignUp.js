import React from 'react';
import {useFormik} from 'formik';
import {Link} from "react-router-dom";
import {NotificationManager, NotificationContainer} from "react-notifications";

// A custom validation function. This must return an object
// which keys are symmetrical to our values/initialValues
const maxLetters = 20;
const errorsLetters = 'Must be' + maxLetters + 'characters or less';
const validate = values => {
    const errors = {};
    if (values.firstName.length > maxLetters) {
        errors.firstName = errorsLetters;
    }

    if (values.lastName.length > maxLetters) {
        errors.lastName = errorsLetters;
    }

    if (!values.email) {
        errors.email = 'Required';
    } else if (!/^[A-Z\d._%+-]+@[A-Z\d.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
        errors.email = 'Invalid email address';
    }

    if (!values.phone) {
        errors.phone = 'Required';
    }
    if (!values.password) {
        errors.phone = 'Required';
    }
    return errors;
};

const SignupForm = () => {
    // Pass the useFormik() hook initial form values, a validate function that will be called when
    // form values change or fields are blurred, and a submit function that will
    // be called when the form is submitted
    const formik = useFormik({
        initialValues: {
            firstName: '',
            lastName: '',
            email: '',
            phone: '',
            password: '',
            category: 'cl'
        },
        validate,
        onSubmit: values => {
            /*
                TODO change with saving information in the database
             */
            alert(JSON.stringify(values, null, 2));
            NotificationManager.success('Signing up successfully!');
        },
    });
    return (
        <form onSubmit={formik.handleSubmit} onReset={formik.resetForm} >
            <label htmlFor="firstName" className="m-2">Nome: </label>
            <input
                id="firstName"
                name="firstName"
                type="text"
                onChange={formik.handleChange}
                value={formik.values.firstName}
            />
            {formik.errors.firstName ? <div className="error">{formik.errors.firstName}</div> : null}
            <br/>
            <label htmlFor="lastName" className="m-2">Cognome: </label>
            <input
                id="lastName"
                name="lastName"
                type="text"
                onChange={formik.handleChange}
                value={formik.values.lastName}
            />
            {formik.errors.lastName ? <div className="error">{formik.errors.lastName}</div> : null}
            <br/>
            <label htmlFor="email" className="m-2">Email: </label>
            <input
                id="email"
                name="email"
                type="email"
                onChange={formik.handleChange}
                value={formik.values.email}
            />
            {formik.errors.email ? <div className="error">{formik.errors.email}</div> : null}
            <br/>
            <label htmlFor="phone" className="m-2">Telefono: </label>
            <input
                id="phone"
                name="phone"
                type="tel"
                onChange={formik.handleChange}
                value={formik.values.phone}
            />
            {formik.errors.phone ? <div className="error">{formik.errors.phone}</div> : null}
            <br/>
            <label htmlFor="password" className="m-2">Password </label>
            <input
                id="password"
                name="password"
                type="password"
                onChange={formik.handleChange}
                value={formik.values.password}
            />
            {formik.errors.password ? <div className="error">{formik.errors.password}</div> : null}
            <br/>
            <label htmlFor="category" className="m-2">Categoria: </label>
            <select
                id="category"
                name="category"
                onChange={formik.handleChange}
                value={formik.values.category}
            >
                <option value="cl">Cliente</option>
                <option value="org">Organizzatore</option>
                <option value="amm">Amministratore</option>
            </select>
            <br/>
            <button type="reset" className="m-2">Reset</button>
            <button type="submit" className="m-2">Submit</button>
            <br/>
            <Link to="/login" className="small m-2">Sei gi√† iscritto?</Link>
            <NotificationContainer />
        </form>
    );
};

export default SignupForm;