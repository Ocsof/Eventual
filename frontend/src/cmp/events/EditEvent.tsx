import {Button, Card} from "react-bootstrap";
import * as React from "react";
import {useState} from "react";
import {EventCardProps} from "./Event";
import {useNavigate} from "react-router-dom";
import { NotificationManager } from "react-notifications";

export function EditEvent({id, title, author, category, date, description, imgUrl}:EventCardProps) {
    const navigate = useNavigate();

    const [event, setEvent] = useState({
        id: id,
        title: title,
        author: author,
        category: category,
        date: date,
        description: description,
        imgUrl: imgUrl
    })

    function handleSave(){
        NotificationManager.success("Event modified!");
        alert(JSON.stringify(event))
        console.log(event);
        navigate(0)
    }

    return (
        <Card>
            <Card.Img
                variant="top"
                src={imgUrl}
                height="200px"
                style={{ objectFit: "cover" }}
            />
            <Card.Body className="d-flex flex-column">
                <Card.Title className="d-flex justify-content-between align-items-baseline">
                    <input style={{width: "50%"}} className="fs-2 m-1" type="text"  placeholder={event.title} value={event.title} onChange={(e) => setEvent({...event, title: e.target.value})} />
                    <input style={{width: "50%"}} className="fs-2 m-1" type="text" placeholder={event.author} value={event.author} onChange={(e) => setEvent({...event, author: e.target.value})} />
                </Card.Title>
                <Card.Text className="mt-auto" >
                    <input style={{width: "100%"}} className="fs-4 m-1" type="text" placeholder={event.category} value={event.category} onChange={(e) => setEvent({...event, category: e.target.value})} />
                    <input style={{width: "100%"}} className="fs-4 m-1" type="text" placeholder={event.description} value={event.description} onChange={(e) => setEvent({...event, description: e.target.value})} />
                    <input className="m-1" type="date" value={event.date} onChange={(e) => setEvent({...event, date: e.target.value})} />
                </Card.Text>
                <Card.Footer >
                    <Button style={{marginLeft: "90%"}} className="btn btn-success" onClick={handleSave}>Save</Button>
                </Card.Footer>
            </Card.Body>
        </Card>
    )
}