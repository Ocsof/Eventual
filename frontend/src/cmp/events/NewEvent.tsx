import {Button, Card} from "react-bootstrap";
import * as React from "react";
import {useState} from "react";
import {useNavigate} from "react-router-dom";
import { NotificationManager } from "react-notifications";

export function NewEvent() {
    const navigate = useNavigate();

    const [event, setEvent] = useState({
        id: -1,
        title: '',
        author: '',
        category: '',
        date: '',
        description: '',
        imgUrl: ''
    })

    function handleSave(){
        NotificationManager.success("Event created");
        console.log(event);
        navigate("/events");
    }

    return (
        <Card className="m-4 h-100 w-75">
            <Card.Img
                variant="top"
                src={event.imgUrl}
                height="200px"
                style={{ objectFit: "cover" }}
            />
            <Card.Title>
                <h3>New Event</h3>
            </Card.Title>
            <Card.Body className="d-flex flex-column">
                <label className="form-label" htmlFor="newEventImg">Event image: </label>
                <input style={{width: "100%"}} type="file" className="form-control" id="newEventImg" onChange={(e) => setEvent({...event, imgUrl: e.target.value})}/>
                <label className="form-label" htmlFor="newEventTitle">title: </label>
                <input style={{width: "100%"}} type="text" className="form-control" id="newEventTitle" onChange={(e) => setEvent({...event, title: e.target.value})}/>
                <label className="form-label" htmlFor="newEventAuthor">author: </label>
                <input style={{width: "100%"}} type="text" className="form-control" id="newEventAuthor" onChange={(e) => setEvent({...event, author: e.target.value})}/>
                <label className="form-label" htmlFor="newEventCategory">category: </label>
                <input style={{width: "100%"}} type="text" className="form-control" id="newEventCategory" onChange={(e) => setEvent({...event, category: e.target.value})}/>
                <label className="form-label" htmlFor="newEventDate">date: </label>
                <input style={{width: "100%"}} type="date" className="form-control" id="newEventDate" onChange={(e) => setEvent({...event, date: e.target.value})}/>
                <label className="form-label" htmlFor="newEventDesc">description: </label>
                <input style={{width: "100%"}} type="text" className="form-control" id="newEventDesc" onChange={(e) => setEvent({...event, description: e.target.value})}/>

                <Card.Footer >
                    <Button style={{marginLeft: "90%"}} className="btn btn-success" onClick={handleSave}>Save</Button>
                </Card.Footer>
            </Card.Body>
        </Card>
    )
}