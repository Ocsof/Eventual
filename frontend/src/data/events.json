[
  {
    "id": 1,
    "title": "Event 1",
    "author": "test",
    "category": "concert",
    "date": "2000-01-01",
    "description": "description concert",
    "imgUrl": "/img/event1.jpg"
  },
  {
    "id": 2,
    "title": "Event 2",
    "author": "test",
    "category": "party",
    "date": "2000-01-01",
    "description": "description party",
    "imgUrl": "/img/event2.jpg"
  },
  {
    "id": 3,
    "title": "Event 3",
    "author": "Ale",
    "category": "private event",
    "date": "2023-01-01",
    "description": "description private event",
    "imgUrl": "/img/event3.jpg"
  }
]