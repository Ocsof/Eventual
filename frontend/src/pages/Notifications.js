import React from 'react';

const Notifications = () => {
    return (
        <div>
            <p>Future feature: Notifications</p>
            <p>Try the chat <a href="http://localhost:3001/chat">here</a></p>
        </div>
    );
};

export default Notifications;