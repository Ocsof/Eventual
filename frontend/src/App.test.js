import { render, screen } from '@testing-library/react';
import Notifications from "./cmp/Notifications";
import {NewEvent} from "./cmp/events/NewEvent";
import {BrowserRouter, Link} from "react-router-dom";
import SignUp from "./cmp/access/SignUp";
import renderer from 'react-test-renderer';
import {Login} from "./cmp/access/Login";
import {EditEvent} from "./cmp/events/EditEvent";
import events from "./data/events.json";

Object.defineProperty(window, 'matchMedia', {
    writable: true,
    value: jest.fn().mockImplementation(query => ({
        matches: false,
        media: query,
        onchange: null,
        addListener: jest.fn(), // Deprecated
        removeListener: jest.fn(), // Deprecated
        addEventListener: jest.fn(),
        removeEventListener: jest.fn(),
        dispatchEvent: jest.fn(),
    })),
});

test('Application Routes renders correctly', () => {
    const tree = renderer
        .create(
            <BrowserRouter>
                <Link to="/">Home</Link>
                <Link to="/new_event">NewEvent</Link>
                <Link to="/login">Login</Link>
                <Link to="/admin">Admin</Link>
            </BrowserRouter>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});

it('Signup component is loaded', () =>{

    render(
        <BrowserRouter>
            <SignUp />
        </BrowserRouter >
    );

    const element = screen.getByText(/Sign/i);
    expect(element).toBeInTheDocument();
});

it('Login component is loaded', () =>{

    render(
        <BrowserRouter>
            <Login />
        </BrowserRouter >
    );

    const element = screen.getByText(/Login/i);
    expect(element).toBeInTheDocument();
});

it('EditEvent component is loaded', () =>{
    const title = events[0].title; //Event 1
    render(
        <BrowserRouter>
            <EditEvent {...events[0]} />
        </BrowserRouter>
    );

    const element = screen.getByText(/Event 1/i);
    expect(element).toBeInTheDocument();
});

it('Notifications component is loaded', () => {
    render(<Notifications />);

    const element = screen.getByText(/Notifiche/i);

    expect(element).toBeInTheDocument();
});

it('NewEvent is loaded', () =>{

    render(
        <BrowserRouter>
            <NewEvent />
        </BrowserRouter >
    );

    const element = screen.getByText(/New Event/i);
    expect(element).toBeInTheDocument();
});

it('Signup component is loaded', () =>{

    render(
        <BrowserRouter>
            <SignUp />
        </BrowserRouter >
    );

    const element = screen.getByText(/Sign/i);
    expect(element).toBeInTheDocument();
});


