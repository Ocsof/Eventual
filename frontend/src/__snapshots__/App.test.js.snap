// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application Routes renders correctly 1`] = `
Array [
  <a
    href="/"
    onClick={[Function]}
  >
    Home
  </a>,
  <a
    href="/new_event"
    onClick={[Function]}
  >
    NewEvent
  </a>,
  <a
    href="/login"
    onClick={[Function]}
  >
    Login
  </a>,
  <a
    href="/admin"
    onClick={[Function]}
  >
    Admin
  </a>,
]
`;
